stdlib/2/os/__init__.pyi

```diff
@@ -3,6 +3,7 @@
 
 from builtins import OSError as error
 from io import TextIOWrapper as _TextIOWrapper
+from posix import stat_result as stat_result  # TODO: use this, see https://github.com/python/mypy/issues/3078
```

Comment:
In Python 3 stats_result only exists in `os` but not `posix`. I'll fix it later.